### DISCUSSION TITLE
Total Size and User Tags

### DISCUSSION THREAD

**Discussion by atianpup** [sct/overseerr#3313] [Feature Requests]
Me and my Friends,

have a feature we think would benefit a lot of people. To see Users Total size of everything they requested, and also every Request individual size. 

I’m also missing a feature I think would be nice, tags automatically added to a User, that way in Sonarr and Radarr you can see the tags. And also for it to work with Plex Watchlist, to auto add the tags. And not allow a user to change / chose a tag unless they have permissions for it.

I hope these features can be a part of Overseerr because it would really make it a lot easier to manage as the owner.

**Comment by IzaacJ** [comment]
I'll take a look at this if work allows for it, since it's a feature I'd love to see as well! I hope I can make time for it this week :)

**Reply by atianpup** [reply]
That would be Amazing! Thank you! ^^

**Reply by IzaacJ** [reply]
If I understood your text correctly, you wanted usernames tagged on the requests, right? I got that working and have made a pull request with that :)

https://github.com/sct/overseerr/pull/3338

**Reply by atianpup** [reply]
That's amazing! ^^ Thank you
